MoviesMVVM/MoviesMVVM/MainModule/View/MoviesViewController.swift

```diff
@@ -17,6 +19,7 @@ final class MoviesViewController: UIViewController {
         static let movies = "Фильмы"
         static let keyValue = "a5b0bb6ebe58602d88ccf2463076122b"
```

Comment:
```suggestion
        static let keyValue = "a5b0bb6ebe58602d88ccf2463076122b"
         static let key = "apiKey"
```
что это делает в контролере